domains/zeusgang.1bt.uk.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "Browser extension Random User-Agent homepage",
```

Comment:
Is that the correct description?

Reply:
nah I'm sorry, I actually copied someone else's, but for the website it's just the team's personal website

Reply:
and i already starred this repo

Reply:
change this line to something relevant, please